pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -417,9 +423,11 @@ func (p *PriorityQueue) AddUnschedulableIfNotPresent(pInfo *framework.QueuedPodI
 		if err := p.podBackoffQ.Add(pInfo); err != nil {
 			return fmt.Errorf("error adding pod %v to the backoff queue: %v", klog.KObj(pod), err)
 		}
+		klog.V(5).InfoS("Pod moved", "pod", klog.KObj(pod), "event", ScheduleAttemptFailure, "from", activeQName, "to", backoffQName)
```

Comment:
why does it say from active? Before calling these functions, the pod was out of the queues.

Reply:
Added a new constant `notInQueue` and changed  `from` to `notInQueue`